lib/cmd/fh3/credential/create.js

```diff
@@ -0,0 +1,105 @@
+/* globals i18n */
+var fhreq = require('../../../utils/request.js');
+var fs = require('fs');
+
+module.exports = {
+  'desc' : i18n._('Create a credential bundle'),
+  'examples' : [{
+    cmd : 'fhc credential create --bundleName=<bundle-name> --bundleType=<bundle-type> --platform=<platform> --env=<env> --privateKey=<private-key> --certificateAndroid=<certificate-android>',
+    desc : 'Create a credential for Android',
+  },
+  {
+    cmd : 'fhc credential create --bundleName=<bundle-name> --bundleType=<bundle-type> --platform=<platform> --env=<env>  --privateKey=<private-key> --certificateIos=<certificate-ios> --provisioning=<provisioning>',
+    desc : 'Create a credential for IOS',
+  }],
+  'demand' : ['bundleName', 'platform'],
+  'alias' : {},
+  'describe' : {
+    'bundleName' : i18n._('Name for the credential which you want create.'),
+    'bundleType': i18n._("FThe default value is 'release'. Option valid just for IOS platform. Options :['release','debug', 'distribution']"),
+    'platform': i18n._("Platform Options : [android, ios]"),
+    'env': i18n._("Default value is all. Environment ID which you want create the credential."),
+    'privateKey': i18n._("Param valid for IOS and Android platform. A Private Key is a file whose contents are secret to the owner. During the App building process, the App is digitally signed using this key. Please make sure that Private Key is password protected. Path for the private key file (E.g --privateKey:/Users/user/rivateKey)"),
+    'certificateAndroid': i18n._("Param valid for Android platform. A Distribution Certificate is used to build Release Apps for submission to the Android Marketplace. When you submit an App to the Android Marketplace, your certificate is used to identify you as the developer of the App. Path for the certificate file. (E.g --certificate:/Users/user/certificate)"),
+    'certificateIos': i18n._("Param valid for IOS platform. Your iOS developer certificate file issued by Apple. You can download this file from your iOS developer portal. Path for the certificate file. (E.g --certificate:/Users/user/certificate)"),
+    'provisioning': i18n._("Param valid for IOS platform. A provisioning profile for a particular App ID downloaded from the iOS developer portal. The type of the provisioning profile should match the type selected above. A Distribution Certificate is used to build Release Apps for submission to the Android Marketplace. When you submit an App to the Android Marketplace, your certificate is used to identify you as the developer of the App Path for the provisioning file. (E.g --provisioning:/Users/user/provisioning)")
+  },
+  'url' : '/box/api/credentials',
+  'method' : 'post',
+  'customCmd' : function(params,cb) {
+    if (params.platform && params.platform !== 'ios' && params.platform !== 'android') {
```

Comment:
This change doesn't improve the performance, the effort to make the operations are the same and in my point of view the "if" make the understanding of the code easier for other developers. So, in this way, I will keep this one in this way.